src/main/kotlin/com/msd/robot/domain/UpgradeException.kt

```diff
@@ -1,5 +1,8 @@
 package com.msd.robot.domain
 
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.ResponseStatus
 import java.lang.RuntimeException
 
+@ResponseStatus(HttpStatus.BAD_REQUEST)
```

Comment:
This should be a `409`, since this exception covers the "Upgrade not possible" responses.